### PR TITLE
Feature/crate animals table

### PR DESCRIPTION
**Pull request by OmarRamoun**
# Changes Introduced ♻️ 

## ✔️ Create a table named animals with the following columns:
- id
- name
- date_of_birth
- escape_attempts
- neutered
- weight_kg

## ✔️ Insert the following data:
```sql
 id |  name   | date_of_birth | escape_attempts | neutered | weight_kg
----+---------+---------------+-----------------+----------+-----------
  5 | Agumon  | 2020-02-03    |               0 | t        |     10.23
  6 | Gabumon | 2018-11-15    |               2 | t        |         8
  7 | Pikachu | 2021-01-07    |               1 | f        |     15.04
  8 | Devimon | 2017-05-12    |               5 | t        |        11
```

## ✔️ Write queries for the following:

- **Find** all animals whose name ends in "mon".
- **List** the name of all animals born between 2016 and 2019.
- **List** the name of all animals that are neutered and have less than 3 escape attempts.
- **List** date of birth of all animals named either "Agumon" or "Pikachu".
- **List** name and escape attempts of animals that weigh more than 10.5kg
- **Find** all animals that are neutered.
- **Find** all animals not named Gabumon.
- **Find** all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+POSTGRESQL
+Create a table named animals with the following columns:
+    id: integer
+    name: string
+    date_of_birth: date
+    escape_attempts: integer
+    neutered: boolean
+    weight_kg: decimal
+*/
+
```
Comment (Mihndim2020) [inline schema.sql]: - [x] Please kindly remove commented code from your code base as it makes your code base dirty and less readable.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @MrRamoun, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (OmarRamoun) [reply schema.sql]: Thank You @Mihndim2020

**Review by CollinsTatang — APPROVED** [review]
Hi @MrRamoun, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**